Sources/BraintreeShopperInsights/BTPresentmentDetails.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+///  `BTPresentmentDetails` Configure detailed information about the presented button.
+/// - Warning: This class is in beta.
```

Comment:
To match the rest of the SDK:
```suggestion
/// - Warning: This class is in beta. It's public API may change or be removed in future releases.
```